### PR TITLE
#111: wire up material theme colors

### PR DESCRIPTION
**Pull request by rpeterman-gp**
- add sass functions for theming
- replace static colors with theme variables or theme map prop
- adjustment made to campaign form layout
- flight table paginator should not scroll with overflowed cells

### To Review

- [ ] Checkout branch.
- [ ] Run `yarn`.
- [ ] Run `docker-compose build`.
- [ ] Run `docker-compose up`.
- [ ] Start `http-proxy`.
- [ ] Go to [grove.prx.docker](https://grove.prx.docker).
- [ ] Ensure theme coloring is as expected.
- [ ] Edit `/sass/_variables.scss` and change the function used on line 13 to `mat-dark-theme`.
- [ ] Ensure dark theme is applied as expected (this is to demonstrate the theme variables are wired up correctly.)
- [ ] Go to Flights table and adjust screen so the table can scroll horizontally.
- [ ] Ensure paginator stays in place as you scroll the table.
- [ ] Go to a campaign and note the fields layout with horizontal space between fields when they are in two columns.

### CODE REVIEW

File: src/app/dashboard/flight-table/flight-table.component.scss [hunk 1]
```diff
@@ -82,3 +82,8 @@ $cell-padding: $cell-padding-v $cell-padding-h;
     color: $warn-contrast;
   }
 }
+
```
Comment (sandikbarr) [inline src/app/dashboard/flight-table/flight-table.component.scss:85]: what's this change for?

**Review by sandikbarr — CHANGES_REQUESTED** [review]
looks really great, just one question on that sticky paginator change. thanks for doing this!

File: src/app/dashboard/flight-table/flight-table.component.scss (hunk 1 above)
Comment (rpeterman-gp) [reply src/app/dashboard/flight-table/flight-table.component.scss:85]: I noticed when the table scrolled horizontally, the paginator would also move. Currently it scrolls almost halfway across the container, but if the table were to contain more columns or wider column content, could end up scrolling to a point some  of the buttons could be come inaccessible. This change makes the paginator stay in one place when the table is scrolled horizontally.

Comment (sandikbarr) [reply src/app/dashboard/flight-table/flight-table.component.scss:85]: good demo, makes sense. thanks!

**Review by sandikbarr — APPROVED** [review]
looks great!

**Pull request merged**